src/app.js

```diff
@@ -21,4 +21,4 @@ app.use((req, res, next) => {
 
 app.listen(process.env.PORT || 3000, () => {});
 
-module.exports = app; // for testing
+export { app };
```

Comment:
I think this is supposed to be `export default app`. Please read through the documentation to understand the difference